src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeAuthenticationPage.java

```diff
@@ -0,0 +1,19 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class AutomationPracticeAuthenticationPage extends AbstractPage {
```

Comment:
What is the difference between this page and AutomationPracticeSignInPage?

Reply:
The only difference is that in sign in you dont have a purchase summary and you do have in the authentication. But there is no other difference, so I think I could reuse the same page, right?

Reply:
yes, I think you can reuse it